src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write code here
+const listContainer = document.querySelector('ul');
+
+const sortList = (list) => {
+  const people = [...list.querySelectorAll('li')];
```

Comment:
```suggestion
const list = [...document.querySelectorAll('li')];

const sortList = (list) => {
```
мала на увазі якось так)